client/src/models/enums/IdentificationTypesCodes.ts

```diff
@@ -0,0 +1,9 @@
+enum IdentificationTypesCodes {
+    ID=1,
+    PASSPORT=2,
+    OTHER=3,
+    MOSSAD=4,
+    PALESTINE_ID=5
+};
```

Comment:
If we're getting it from the server why keep a local record of all of the types? think about what happens if you want to add another ID type.  
I don't ask you to change it just to rethink about the system's structure :\

Reply:
I completely agree with you, the reason we added enum is that we needed access to the identification codes in a file that don't allow use of selector

Reply:
🤷